server/src/main/java/io/crate/planner/statement/CopyToPlan.java

```diff
@@ -115,24 +124,21 @@ public void executeOrFail(DependencyCarrier executor,
 
         NodeOperationTree nodeOpTree = NodeOperationTreeGenerator
             .fromPlan(executionPlan, executor.localNodeId());
-        executor.phasesTaskFactory()
-            .create(plannerContext.jobId(), List.of(nodeOpTree))
-            .execute(consumer, plannerContext.transactionContext());
+
+        JobLauncher jobLauncher = executor.phasesTaskFactory()
+            .create(plannerContext.jobId(), List.of(nodeOpTree));
+
+        JobLauncher.execute(consumer, plannerContext.transactionContext(), jobLauncher::execute,
+                            boundedCopyTo.withClauseOptions().getAsBoolean("wait_for_completion", true));
```

Comment:
nitpick/minor code style suggestion/preference as this increases readability and produces better diffs on possible future changes. our general pattern here is, either all in one line if fitting into our max char of 120, or each arg in a newline.
```suggestion
        JobLauncher.execute(
            consumer,
            plannerContext.transactionContext(),
            jobLauncher::execute,
            boundedCopyTo.withClauseOptions().getAsBoolean("wait_for_completion", true)
        );
```